src/util/getEventHandlers.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path')
+const glob = require('glob')
+
+const messageModule = require.resolve('~/src/messages')
+const messagesDir = messageModule.substring(0, messageModule.length - '/index.js'.length)
+
+const eventHandlers = {}
+let initialized = false
+
+module.exports = function getEventHandlers () {
+  if (!initialized) {
+    // walk events dir to get list of event handlers
+    const eventHandlerFiles = glob.sync(
+      path.resolve(messagesDir, 'events/**/*.js'))
```

Comment:
I did this so that we can try to maintain some sort of organization when adding different message handlers. Ideally the placement of handlers should match up with the structure of our Event types in the service util repo.